Modules/Announcements/src/Announcements.Application/Announcements/Commands/DeleteAnnouncementById/DeleteAnnouncementByIdCommand.cs

```diff
@@ -0,0 +1,8 @@
+using MediatR;
+
+namespace Backbone.Modules.Announcements.Application.Announcements.Commands.DeleteAnnouncementById;
+
+public class DeleteAnnouncementByIdCommand(string id) : IRequest
+{
+    public string Id { get; } = id;
+}
```

Comment:
I'd suggest the following:

```suggestion
public class DeleteAnnouncementByIdCommand : IRequest
{
    public required string Id { get; }
}
```

And I know we're not consistent with how we handle this. But imo this is the nicest way to do it.